taskcluster/ci/ui-test-apk/kind.yml

```diff
@@ -170,18 +170,96 @@ tasks:
+                - [python3, automation/taskcluster/androidTest/testrail.py]
         treeherder:
             platform: 'focus-ui-test/opt'
             symbol: focus-beta(ui-test-arm-beta)
         worker:
             env:
                 GOOGLE_PROJECT: 'moz-focus-android'
+                PRODUCT_TYPE: Focus
+                RELEASE_TYPE: Beta
+        routes:
+            - notify.slack-channel.G016BC5FUHJ.on-completed
+        scopes:
+            - queue:route:notify.slack-channel.G016BC5FUHJ.on-completed
+                - notify:slack-channel:G016BC5FUHJ
+        extra:
+            notify:
+                # slackText: 'https://firefox-ci-tc.services.mozilla.com/tasks/${status.taskId} | ${task.metadata.name} | ${task.metadata.source}'
+                slackBlocks: |
+                    [
+                      {
+                        "type": "header",
+                        "text": {
+                          "type": "plain_text",
+                          "text": "firefox-android :tada: Testrail Release Milestone Creation :tada:\n "
+                        }
+                      },
+                      {
+                        "type": "divider"
+                      },
+                      {
+                        "type": "section",
+                        "text": {
+                            "type": "mrkdwn",
+                            "text": "*Task*: <https://firefox-ci-tc.services.mozilla.com/tasks/${status.taskId}|Taskcluster>"
+                        }
+                      },
+                      {
+                        "type": "section",
+                        "text": {
+                          "type": "mrkdwn",
+                          "text": "*Owner*: ${task.metadata.owner}"
+                        }
+                      },
+                      {
+                        "type": "section",
+                        "text": {
+                          "type": "mrkdwn",
+                          "text": "*Source*: <${task.payload.env.MOBILE_BASE_REPOSITORY}/commit/${task.payload.env.MOBILE_HEAD_REV}|Commit> :github:"
+                        }
+                      },
+                      {
+                        "type": "divider"
+                      },
+                      {
+                        "type": "section",
+                        "text": {
+                          "type": "mrkdwn",
+                          "text": "*Testrail Release Milestone*: <https://testrail.stage.mozaws.net/index.php?/projects/overview/53|Milestone> :clown_face:"
+                        }
+                      },
+                      {
+                        "type": "section",
+                        "text": {
+                            "type": "mrkdwn",
+                            "text": "Hello <@Jackie> and <@rpapa>, *Testrail Release Milestone* has been created"
```

Comment:
Should this be `@mobiletest`?

Reply:
removed it, if we want to `@` someone, we'll need to use their slack `UUID`. we can use `@mobiletest` if we want. might be too much noise during phase 1 testing.